performanceplatform/client/base.py

```diff
@@ -115,8 +115,10 @@ def _request(self, method, path, data=None):
             except:
                 log.error('[PP-C] {}'.format(response.text))
                 raise
-
-            json = response.json()
+            try:
+                json = response.json()
+            except ValueError:
```

Comment:
Does this mean we ignore invalid json and assume it is blank? Can we check if the response is empty and then pass, otherwise raise?